src/components/EntryList/EntryList.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import EntryList from './EntryList';
+import { screen, render } from '@testing-library/react';
+import { EntryProvider } from '../../context/EntryContext';
+
+it('renders list of entries', async () => {
+  const { container } = render(
+    <EntryProvider>
+      <EntryList />
+    </EntryProvider>
+  );
+  expect(container).toMatchSnapshot;
```

Comment:
.toMatchSnapshot needs to be a function in order for it to actually take the snapshot. Make sure the parentheses are added to this and it runs locally, creating a snapshot